src/modules/rppt_tensor_effects_augmentations.cpp

```diff
@@ -989,4 +1090,92 @@ RppStatus rppt_non_linear_blend_gpu(RppPtr_t srcPtr1,
 #endif // backend
 }
 
+/******************** glitch ********************/
+
+RppStatus rppt_glitch_gpu(RppPtr_t srcPtr,
+                          RpptDescPtr srcDescPtr,
+                          RppPtr_t dstPtr,
+                          RpptDescPtr dstDescPtr,
+                          Rpp32u *xOffsetR,
+                          Rpp32u *yOffsetR,
+                          Rpp32u *xOffsetG,
+                          Rpp32u *yOffsetG,
+                          Rpp32u *xOffsetB,
+                          Rpp32u *yOffsetB,
+                          RpptROIPtr roiTensorPtrSrc,
+                          RpptRoiType roiType,
+                          rppHandle_t rppHandle)
+{
+#ifdef HIP_COMPILE
+    if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::U8))
+    {
+        hip_exec_glitch_tensor(static_cast<Rpp8u*>(srcPtr) + srcDescPtr->offsetInBytes,
+                               srcDescPtr,
+                               static_cast<Rpp8u*>(dstPtr) + dstDescPtr->offsetInBytes,
+                               dstDescPtr,
+                               xOffsetR,
+                               yOffsetR,
```

Comment:
No need to resend the parameters xOffsetR, xOffsetG, xOffsetB yOffsetR, yOffsetG, yOffsetB
since they will be automatically copied to device memory in handle by using copy_param_uint

Reply:
Done